src/main.rs

```diff
@@ -86,8 +89,16 @@ async fn run(cfg: Config, inputs: Vec<String>) -> Result<i32> {
     } else {
         None
     };
+
+    if cfg.github_token.is_none() {
```

Comment:
Guess the default use-case would be non-Github links so I'd remove that warning as it might be confusing. How about we extend the status message from above?
Right now it is
```rust
Status::Error("GitHub token not specified".to_string()),
```
How about
```rust
Status::Error("GitHub token not specified. Use `--github-token` flag / `GITHUB_TOKEN` env var.".to_string()),
```